pgx_unix/src/pgx_unix.ml

```diff
@@ -54,6 +54,8 @@ module Simple_thread = struct
   let input_binary_int = input_binary_int
   let really_input = really_input
   let close_in = close_in
+
+  (* The unix getlogin syscall can fail *)
```

Comment:
The reason we don't use getlogin here is just because `Async.Unix.getlogin` is a blocking function

Reply:
The Lwt_unix of getlogin is also just calling Unix.getlogin:
```
let getlogin () =
  if Sys.win32 || Lwt_config.android then
    Lwt.return (Unix.getlogin ())
  else
    run_job (getlogin_job ())
```

Reply:
Ah ok